docs/code.rst

```diff
@@ -120,7 +120,9 @@ Variable                            Description
 =================================   ====================================
 ``{{ ds }}``                        the execution date as ``YYYY-MM-DD``
 ``{{ prev_ds }}``                   the previous execution date as ``YYYY-MM-DD``
+                                    and it resolves to ``None`` when ``schedule_interval="@once"``
```

Comment:
Or if there is no previous execution date, right? e.g., the first DagRun will have no previous execution date. Maybe we should just say something like `the previous execution date as ``YYYY-MM-DD`` if it exists, else None`?

Reply:
That's better.

Reply:
addressed.